src/DOMTestCase.php

```diff
@@ -149,7 +236,27 @@ public static function assertSelectEquals($selector, $content, $count, $actual,
         }
 
         else {
-            throw new PHPUnit_Framework_Exception('Invalid count format');
+            throw new PHPUnit_Framework_Exception('Invalid count format. ' . $count);
         }
     }
+
+    /**
+     * @param Crawler $crawler
+     * @param $content
+     * @return Crawler
+     */
+    private static function filterCrawler(Crawler $crawler, $content)
+    {
+        return $crawler->reduce(function (Crawler $node, $i) use ($content) {
+            if ($content === '') {
+                return $node->text() === '';
+            }
+
+            if (preg_match('/^regexp\s*:\s*(.*)/i', $content, $matches)) {
```

Comment:
You can use named matches like this to make it more readable:

```
/^regexp\s:*\s*(?P<matchName>.*)/i
```

That allows you to access it by `$matches['matchName']`